Common Utilities/Patches/Scp049SpeechPatch.cs

```diff
@@ -1,22 +1,40 @@
 using Assets._Scripts.Dissonance;
 using HarmonyLib;
 
-namespace Common_Utilities.Patches
+namespace VoiceChatSCP
```

Comment:
There's no reason to change the namespace here

Reply:
my bad forgot to change hat after copying it over